projects/storefrontlib/src/lib/cms/cms-module-config.ts

```diff
@@ -34,6 +41,7 @@ export abstract class CmsModuleConfig extends OccConfig
   };
 
   cmsComponentMapping?: CMSComponentMappingConfig;
+  cmsComponentProviders?: CMSComponentProviders;
```

Comment:
One of the ideas that I had earlier this week (forgot to tell), is to combine the configuration of the component mapping and component providers in one object. The config would have a typing that allows for both as an optional property. I believe it simplifies configuration as well as its' all more connected when you customise.